src/asciidoc/docs/guidelines/developerTip.PersistingDataAcrossSessions.adoc

```diff
@@ -0,0 +1,50 @@
+= Developer Tip: Persisting data across sessions
+:page-slug: persisting-data-across-sessions
+:page-description: How to persist data that doesn't expire with the session.
+
+Data can be stored temporarily using window.sessionStorage. However, data stored in this way will expire when the session ends. If data needs to be persisted across sessions, `window.localStorage` may be used. Data stored in this way has no expiration date. It will be available until the browser's cache is cleared.
```

Comment:
Let's add some Koji-specific perspective to this intro. How might a template use localStorage? For an example of a similar page, see https://developer.withkoji.com/docs/guidelines/frontend-uploading

Reply:
Marko supplied me with a simple chatbox template that uses local storage. I added links to the app and the relevant code.

Reply:
The chatbox is a great example to cite, and I like including a link to it. The language needs refinement. In particular, we need to make it a bit more generic and smoother.

Also, avoid "app" in the technical docs. We generally say "template" (for the base application that a developer publishes) and "Koji" (for the remix of a template that a creator publishes).

Reply:
Done